lua/gitblame/init.lua

```diff
@@ -181,15 +182,17 @@ local function check_uses_relative_date()
     return false
 end
 
----@param date osdate
+---@param date integer
 ---@return string
 local function format_date(date)
     local format = get_date_format()
     if check_uses_relative_date() then
         format = format:gsub("%%r", timeago.format(date))
     end
-
-    return os.date(format, date)
+    if format == '*t' then
+        return '*t'
+    end
+    return os.date(format, date) --[[@as string]]
```

Comment:
This is an interesting one. Since we pass in a format string in setup you can technically pass in `*t` and the blame will display the date as a table, so this is a fallback :slightly_smiling_face: This is why we need to assert the return type is string because technically it could be `string|table` but we will check beforehand.

[docs on *t](https://www.lua.org/pil/22.1.html)

Reply:
Oh, this one's interesting!